### PR TITLE
Add forms test projects to main solution

### PR DESCRIPTION
**Pull request by ehuna**
I added the Forms > Example001CSharp test projects to the main MvvmCross\MvvmCross_All.sln solution 

The paths to the nuget packages needed to be fixed, going from '..\..\..\packages' to '..\..\..\..\packages'.

Finally I deleted the project specific Solution - since it is confusing if you open it and the projects don't compile.

Like before, I'm submitting the changes for 'Example001CSharp', once these are approved, I'll work on the others.

I compiled and deployed/ran every project and tested it out, except for the UWP project.

Forms > Example001CSharp > fix PCL nuget paths + add project to main Solution
Forms > Example001CSharp > fix Android nuget paths + add project to main Solution
Forms > Example001CSharp > fix Windows Phone nuget paths + add project to main Solution
Forms > Example001CSharp > fix Windows 8.1 nuget paths + add project to main Solution
Forms > Example001CSharp > fix iOS nuget paths + add project to main Solution
Forms > Example001CSharp > add UWP project to main Solution

### CODE REVIEW

File: MvvmCross_All.sln [hunk 1]
```diff
@@ -407,13 +407,27 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "MvvmCross.Plugins.Visibilit
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "MvvmCross.Plugins.WebBrowser.Uwp", "MvvmCross-Plugins\WebBrowser\MvvmCross.Plugins.WebBrowser.Uwp\MvvmCross.Plugins.WebBrowser.Uwp.csproj", "{08E19EF4-8C86-4F6E-B78A-063983884434}"
 EndProject
-Project("{9344BDBB-3E7F-41FC-A0DD-8665D75EE146}") = "MvvmCross.Platform.Wpf", "MvvmCross\Platform\Wpf\MvvmCross.Platform.Wpf.csproj", "{BDD29D7B-293A-4A4A-8DC8-F18285CBFD72}"
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "MvvmCross.Platform.Wpf", "MvvmCross\Platform\Wpf\MvvmCross.Platform.Wpf.csproj", "{3688D6FC-E7F4-485F-8B27-96C0A8C7EAED}"
```
Comment (martijn00) [inline MvvmCross_All.sln]: How come those id's changed?

File: TestProjects/Forms/Example001CSharp/Example.Android/Example.Android.csproj [hunk 2]
```diff
@@ -20,15 +20,24 @@
...
+    <DebugSymbols>True</DebugSymbols>
     <DebugType>full</DebugType>
     <Optimize>false</Optimize>
     <OutputPath>bin\Debug</OutputPath>
     <DefineConstants>DEBUG;</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
     <ConsolePause>false</ConsolePause>
-    <AndroidLinkMode>None</AndroidLinkMode>
+    <AndroidLinkMode>SdkOnly</AndroidLinkMode>
```
Comment (martijn00) [inline TestProjects/Forms/Example001CSharp/Example.Android/Example.Android.csproj]: I don't think we should enable AndroidLinkMode in debug mode. Only in release.

File: TestProjects/Forms/Example001CSharp/Example.iOS/Example.iOS.csproj [hunk 3]
```diff
@@ -22,8 +22,20 @@
     <WarningLevel>4</WarningLevel>
     <ConsolePause>false</ConsolePause>
     <MtouchArch>i386</MtouchArch>
-    <MtouchLink>None</MtouchLink>
-    <MtouchDebug>true</MtouchDebug>
+    <MtouchLink>SdkOnly</MtouchLink>
```
Comment (martijn00) [inline TestProjects/Forms/Example001CSharp/Example.iOS/Example.iOS.csproj]: Same here. Normally you don't link in debug mode.

File: MvvmCross_All.sln (hunk 1 above)
Comment (ehuna) [inline MvvmCross_All.sln]: I'm not sure, I only added the test projects to the main solution, I'll roll this id change back - nice catch!

File: TestProjects/Forms/Example001CSharp/Example.Android/Example.Android.csproj (hunk 2 above)
Comment (ehuna) [inline TestProjects/Forms/Example001CSharp/Example.Android/Example.Android.csproj]: I was unable to deploy to the Visual Studio Android emulator and run/test it out without enabling 'Sdk only' linking.  

I think it's ok to enable sdk linking - it's more important to have a working test project that works out of the box.

File: TestProjects/Forms/Example001CSharp/Example.iOS/Example.iOS.csproj (hunk 3 above)
Comment (ehuna) [inline TestProjects/Forms/Example001CSharp/Example.iOS/Example.iOS.csproj]: Same issue, I was unable to deploy to the iOS simulator and run/test it out without enabling 'Sdk only' linking.

File: MvvmCross_All.sln (hunk 1 above)
Comment (ehuna) [inline MvvmCross_All.sln]: After taking a look at it, I think we should accept the id change - if you edit the MvvmCross_All.sln , you'll notice that all projects are using the 'FAE04EC0-301F-11D3-BF4B-00C04F79EFBC' id.

It seems that MvvmCross.Platform.Wpf was the only one using the '9344BDBB-3E7F-41FC-A0DD-8665D75EE146' id - and somehow VS figured it out and fixed it.

File: TestProjects/Forms/Example001CSharp/Example.Android/Example.Android.csproj (hunk 2 above)
Comment (ehuna) [inline TestProjects/Forms/Example001CSharp/Example.Android/Example.Android.csproj]: I think it's ok to enable sdk linking - it's more important to have a working test project that works out of the box.

With that said, if you prefer I disable AndroidLinkMode in debug mode, let me know and I'll roll it back.

File: TestProjects/Forms/Example001CSharp/Example.iOS/Example.iOS.csproj (hunk 3 above)
Comment (ehuna) [inline TestProjects/Forms/Example001CSharp/Example.iOS/Example.iOS.csproj]: And same comment: I tihnk it's better to have a working example out of the box.  With that said, if you prefer I disable MTouchLink (SdkOnly) in debug mode, let me know and I'll roll it back.

**Comment by ehuna** [comment]
@martijn00 another idea: we move the forms tests projects to https://github.com/MvvmCross/MvvmCross-Samples - what do you think?

**Comment by martijn00** [comment]
No these are used as test projects for active development, and are not the kind of samples we have in mind for the Samples repo. Could you rebase the latest changes on your branch so we can merge this?

**Comment by ehuna** [comment]
Ah got it, ok no worries.

I've updated the PR - it should merge now.

I do see more GUIDs that were updated in the MvvmCross_All.sln as I worked on the test projects.  The best I can tell is that some projects are missing the 'ProjectGuid' so Visual Studio 2015 is changing GUIDs when the Solution is changed.

http://stackoverflow.com/questions/4194424/visual-studio-keeps-changing-project-sln-file

**Comment by ehuna** [comment]
Thanks for merging this, I'll work on the next sample project now.

**Pull request merged**